packages/quantic/force-app/main/default/lwc/quanticBreadcrumb/quanticBreadcrumb.html

```diff
@@ -0,0 +1,78 @@
+<template>
+    <template if:true={hasBreadcrumbs}>
+        <div class="slds-grid slds-var-m-vertical_x-small">
+            <div class="slds-col slds-col_bump-right no-gutters">
+                <ul>
+                    <template for:each={facetBreadcrumbValues} for:item="breadcrumb">
+                        <li key={breadcrumb.facetId} class="slds-list_horizontal align-baseline">
+                            <span class="nowrap">{breadcrumb.field} : </span>
+                            <ul class="slds-list_horizontal slds-has-block-links_space slds-grid slds-wrap">
+                                <template for:each={breadcrumb.values} for:item="breadcrumbValue">
+                                    <li key={breadcrumbValue.value.value} class="coveo-breadcrumb-value-line nowrap">
+                                        <a onclick={breadcrumbValue.deselect} class="text-hover-underline">{breadcrumbValue.value.value} X</a>
```

Comment:
Out of curiosity, would it be possible to use an icon instead of the `x` character? The breadcrumb values are quite similar to the [SLDS Pill](https://www.lightningdesignsystem.com/components/pills/) component.

Reply:
Yea was reported to me as well, the mockups seem to suggest using the `x` letter instead of an SVG, but I'll change it since it's been twice commented now.

Reply:
I tried using the x icon from slds, it looks weird and I don't have control over the colors :( 

![image](https://user-images.githubusercontent.com/2488155/124945416-1bf87900-dfdc-11eb-8819-7440a1ac6594.png)